students/seoyun/westagram/users/views.py

```diff
@@ -27,10 +29,13 @@ def post(self, request):
             if not re.match(password_check, password):
                 return JsonResponse({'message':'비밀번호 조건을 확인해주세요.'}, status=400)
 
+            hashed_password  = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+            decoded_password = hashed_password.decode('utf-8')
```

Comment:
```suggestion
            hashed_password  = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```
그렇게 복잡하지 않은 코드이고 굳이 활용되지 않을 변수를 지정할 필요는 없어 보이네요.
만약 가독성을 생각하신거라면 다음과 같이 다른 시도를 해보실 수 있습니다.
```suggestion
            hashed_password  = bcrypt.hashpw(
                password.encode('utf-8'), 
                bcrypt.gensalt()
            ).decode('utf-8')
```